rir/src/compiler/pir/type.h

```diff
@@ -326,11 +327,15 @@ struct PirType {
         return PirType(t_.r, flags_ & ~FlagSet(TypeFlags::lazy));
     }
 
-    PirType constexpr forced() const {
+    PirType forced() const {
```

Comment:
please leave this constexpr. you need to use a constexpr to create the type. to add the missing flag it should be enough to just add `...(t_.r | RType::missing, ...)` to the expression that was there